### ISSUE TITLE
Make Animations side panel layout fit more to Firebug

### ISSUE THREAD

**Issue by SebastianZ** [firebug/firebug.next#424]
The timeline within the *Animations* side panel looks a bit ugly and doesn't really fit to the general Firebug layout.

![*Animations* side panel not looking *firebuggy*](https://cloud.githubusercontent.com/assets/958943/10818411/a176f578-7e41-11e5-8686-c5af4e53410e.png)

Tasks:
- Timeline header should be adjusted to fit the column headers in the *Network* panel
- Timeline bars should be adjusted to fit the bars within the *Network* panel
- Adjust fonts within the toolbar
- Adjust play state buttons (#423)

Sebastian

**Comment by compojoom** [comment]
@janodvarko would you please advise me how to fix this?
I have followed the developer guide and now I'm running Firefox with firebug from git. I saw that you've addressed some of the problems in this issue
![ff-animations](https://cloud.githubusercontent.com/assets/693770/10966595/e4840f5e-83b4-11e5-99dc-157257da98bc.PNG)
![fb-animations](https://cloud.githubusercontent.com/assets/693770/10966597/e4857920-83b4-11e5-9be9-69b2db9f2b27.PNG)

But it's actually not fully fixed. When we hover with the mouse over the pause button in near the timeline - the hover state is invisible. 

Also as you can see from the ff-animations screenshot -> the inspector tools is showing a rewind icon. With the browser tools I saw that the markup is there. 
![markup](https://cloud.githubusercontent.com/assets/693770/10966672/671df178-83b5-11e5-8c1c-bd7eb2e4c53c.PNG)

Some of the buttons are falling behind the progressbar and also we don't have any images for those. What are the guidelines here? Where are we taking the images from?

**Comment by SebastianZ** [comment]
You're obviously using the wrong version of Firefox. Ensure that you're running Nightly. Then the *Animations* side panel should look like this:

![*Animations* side panel](https://cloud.githubusercontent.com/assets/958943/10968358/e7c8a858-83c0-11e5-9d7c-4ed5311aef76.png)

Sebastian

**Comment by compojoom** [comment]
@SebastianZ - aaaah. okay! Thanks!

So, it seems that this issue is actually fixed?

**Comment by SebastianZ** [comment]
> So, it seems that this issue is actually fixed?

No, as mentioned above the current style doesn't fit to the general Firebug layout. E.g. Firebug doesn't have striped bars anywhere. So the tasks listed in the [first post](https://github.com/firebug/firebug.next/issues/424#issue-114040619) are still valid.

Sebastian

**Comment by fflorent** [comment]
@compojoom Hello. Thanks for being interested in this issue!

Do you need some help / information of any form?

Florent
